justbar/src/main/java/com/irozon/justbar/BarItem.java

```diff
@@ -44,274 +43,286 @@ public boolean isSelected() {
     @Override
     public void setSelected(boolean selected) {
         this.selected = selected;
-
-        if (selected)
+        if (selected) {
             makeSelected();
-        else
+        } else {
             makeUnSelected();
+        }
+
     }
 
+    /**
+     * This is a constructor of BarItem.
+
```

Comment:
java doc alignment is incorrect. Please check

Reply:
Done